### ISSUE TITLE
CI: failing builds on master

### ISSUE THREAD

**Issue by jreback** [pandas-dev/pandas#29053]
https://travis-ci.org/pandas-dev/pandas/builds/599134469

looks like some issues on 3.8, 3.6 allowed failure builds (i know has been failing a while)

should address these before merging other changes.

cc @pandas-dev/pandas-core

**Comment by jbrockmendel** [comment]
Looks like a new difference between numpy 1.17 vs 1.18:

```
s = pd.Series(
            pd.to_datetime(["NaT", "2000-1-2", "NaT", "2000-1-1", "NaT", "2000-1-3"])
        )

>>> np.minimum.accumulate(s.values)  # numpy 1.17
array([                          'NaT', '2000-01-02T00:00:00.000000000',
       '2000-01-02T00:00:00.000000000', '2000-01-01T00:00:00.000000000',
       '2000-01-01T00:00:00.000000000', '2000-01-01T00:00:00.000000000'],
      dtype='datetime64[ns]')

>>> np.accumulate.minimum(s.values)  # numpy 1.18
array(['NaT', 'NaT', 'NaT', 'NaT', 'NaT', 'NaT'], dtype='datetime64[ns]')
```

**Comment by jbrockmendel** [comment]
https://github.com/numpy/numpy/pull/14717

**Comment by jbrockmendel** [comment]
closed by #29057
